app/assets/javascripts/discourse/app/components/sub-category-item.gjs

```diff
@@ -1,26 +1,31 @@
+import PluginOutlet from "discourse/components/plugin-outlet";
 import CategoryListItem from "discourse/components/category-list-item";
 import CategoryTitleBefore from "discourse/components/category-title-before";
 import CategoryUnread from "discourse/components/category-unread";
 import categoryLink from "discourse/helpers/category-link";
 
 export default class SubCategoryItem extends CategoryListItem {
   <template>
-    {{#unless this.isMuted}}
-      {{#if this.site.mobileView}}
-        {{categoryLink this.category}}
-      {{else}}
-        <span class="subcategory">
-          <CategoryTitleBefore @category={{this.category}} />
-          {{categoryLink this.category hideParent="true"}}
-          {{#unless this.hideUnread}}
-            <CategoryUnread
-              @category={{this.category}}
-              @unreadTopicsCount={{this.unreadTopicsCount}}
-              @newTopicsCount={{this.newTopicsCount}}
-            />
-          {{/unless}}
-        </span>
-      {{/if}}
-    {{/unless}}
+    <PluginOutlet
+      @name="sub-category-item"
+      @outletArgs={{hash subCategoryItem=this}}>
```

Comment:
Exposing an entire component instance to a plugin outlet is not ideal, because it makes the entire component 'public API', and therefore harder to refactor in future.

Please can you update this so that it passes explicit arguments which theme/plugin authors might need.

Also, this `hash` should be swapped out for a `lazyHash`. If you rebase this branch onto latest `main` (or merge `main` into it), then the linting rules should catch that.